index.Rmd

```diff
@@ -89,107 +93,101 @@ Scroll down the license with enter. Accept License (i.e. type yes). Use default
 
 Scroll down the license with enter. Accept License (i.e. type yes). Use default install location (i.e. press Enter). When prompted to prepend the Conda install location to path agree (i.e. type yes). This makes the Python interpreter of Conda the standard Python interpreter and allows you to run Conda commands from command line. -->
 
-This will install Miniconda into `~/miniconda3`. Finally, restart your terminal to be able to use `conda` in the terminal. Next, let's see how to use Conda in case you want make new virtual environments by yourself, or install packages after creating the environment.
+This will install Mamba into `~/mamba`. Finally, restart your terminal to be able to use `mamba` and `conda` in the terminal. Next, let's see how to use Mamba in case you want make new virtual environments by yourself, or install packages after creating the environment.
 
-### Conda usage
-Conda creates isolated environments with sets of packages, that do not interfere with your base Python or with other conda environments. To create an environment:
+## Mamba usage
+Mamba creates isolated `conda` environments with sets of packages, that do not interfere with your base Python or with other `conda` environments. To create an environment:
 
 ```{r, eval=FALSE, engine='bash'}
-conda create --name geotest python=2.7 numpy
+mamba create --name geotest python numpy
 ```
 
-This would create a new environment called _geotest_ with Python 2.7 and _NumPy_ installed into the conda environment. Another option is to create an environment from a `.yaml` file, in which all required modules are listed; see the example later on. To create an environment from such a file, use the argument `--file` (or `--f` in short), see the conda (documentation)[https://docs.conda.io/projects/conda/en/latest/commands/create.html]. To list the available environments:
+This would create a new environment called `geotest` with Python 2.7 and `NumPy` and `Spyder` installed into the `conda` environment. Another option is to create an environment from a `.yaml` file, in which all required modules are listed. You will see an example of this later on in this tutorial. To create an environment from such a file, you can use the argument `--file` (or `--f` in short). 
+
+Let's first list the currently available environments:
 
 ```{r, eval=FALSE, engine='bash'}
-conda info --envs
+mamba info --envs
 ```
 
-Conda puts an asterisk (*) in front of the active environment. To activate the environment:
+Mamba puts an asterisk (*) in front of the active environment. To activate the environment:
 
 ```{r, eval=FALSE, engine='bash'}
-## Cross-platform (but not always working, like in our VDI, try it)
```

Comment:
It indeed does not work by default. It would be good to mention this in some way. If you try it, it will complain that `conda init` was not run, but doing so does fairly invasive changes to the `~/.bashrc` file that are not needed, because `source` works fine.